application/src/pages/Car/Car.test.tsx

```diff
@@ -0,0 +1,46 @@
+import { render } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import { BrowserRouter } from 'react-router-dom';
+import { Car } from './Car';
+import cars from '../../data/database.json';
+import { store } from '../../bll/store';
+import i18n from '../../utils/i18next';
+import { CartPropTypes } from '../../bll/reducers/cartReducer';
+
+let car: CartPropTypes;
+
+beforeEach(() => {
+  car = { ...cars[0], amount: 0, total: 0 };
+});
+const setup = () => {
+  const { getAllByAltText, getByText } = render(
+    <BrowserRouter>
+      <Provider store={store}>
+        <Car
+          picture={car.picture}
+          brand={car.brand}
+          model={car.model}
+          price={car.price}
+          color={car.color}
+          length={car.length}
+          height={car.height}
+          width={car.width}
+          description={car.description}
+          id={car.id}
+        />
+      </Provider>
+    </BrowserRouter>
+  );
+  return [getAllByAltText, getByText];
+};
+describe('Car componenet', () => {
+  test('check only text render', () => {
+    const [getAllByAltText, getByText] = setup();
```

Comment:
I guess it is better not to include the render method to the setUp() function. Because, in future, you may want to get different methods from render().

Reply:
fixed with function setUp